### PR TITLE
Update persistent-volumes.md

### PR DESCRIPTION
**Pull request by jemaltahir**
- Added a section on how to expand existing persistent volume
- https://csc-guide-preview.rahtiapp.fi/origin/jemaltahir-patch-2/cloud/pouta/persistent-volumes/

### CODE REVIEW

File: docs/cloud/pouta/persistent-volumes.md [hunk 1]
```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
...
     openstack volume transfer request accept <transferID> <authKey>
+
+## Expanding size to the attached volumes in the Pouta web interface
+
+Previuosly you have created and attached a volume. In this section you are going to enlarge the size of the volume attached to the instance. Before you attempt for volume expansion you have to detach the volume from the instance, please remember to unmount the volume before detaching it!
+
+    sudo umount /dev/vdb
+
+To expand the volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** buttonn for the volume you want to enlarge and select **Extend Volume**. Input the the volume amount you want to enlarge in (GiB) in the field **New Size (GiB)**. Finally, click the **Extend Volume** button.
+To attach an expanded volume similar to the previous attach persisitent volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** button for the volume you expanded and select **Manage attachments**. Select the instance (i.e. virtual machine) you want to attach the volume to in the **Attach to Instance** selector.
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:194]:
```suggestion
To attach an expanded volume similar to the previous attach persistent volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** button for the volume you expanded and select **Manage attachments**. Select the instance (i.e. virtual machine) you want to attach the volume to in the **Attach to Instance** selector.
```

File: docs/cloud/pouta/persistent-volumes.md [hunk 2]
```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
...
 
     openstack volume transfer request accept <transferID> <authKey>
+
+## Expanding size to the attached volumes in the Pouta web interface
+
+Previuosly you have created and attached a volume. In this section you are going to enlarge the size of the volume attached to the instance. Before you attempt for volume expansion you have to detach the volume from the instance, please remember to unmount the volume before detaching it!
+
+    sudo umount /dev/vdb
+
+To expand the volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** buttonn for the volume you want to enlarge and select **Extend Volume**. Input the the volume amount you want to enlarge in (GiB) in the field **New Size (GiB)**. Finally, click the **Extend Volume** button.
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:193]:
```suggestion
To expand the volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** button for the volume you want to enlarge and select **Extend Volume**. Input the the volume amount you want to enlarge in (GiB) in the field **New Size (GiB)**. Finally, click the **Extend Volume** button.
```

File: docs/cloud/pouta/persistent-volumes.md [hunk 3]
```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
...
+To attach an expanded volume similar to the previous attach persisitent volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** button for the volume you expanded and select **Manage attachments**. Select the instance (i.e. virtual machine) you want to attach the volume to in the **Attach to Instance** selector.
+
+![Expand persistent volume](/img/volume-expand-horizon1.png)
+
+Once you have logged in to your virtual machine, you can list the
+volumes:
+
+    sudo parted -l
+
+Similar to the previous persisitent volume creation you can identify the volume based on its size. First mount the path:
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:203]:
```suggestion
Similar to the previous persistent volume creation you can identify the volume based on its size. First mount the path:
```

File: docs/cloud/pouta/persistent-volumes.md [hunk 4]
```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
...
+
+    sudo mount /dev/vdb /media/volume
+
+Now you can check the status of the volume in a mounted XFS filesystem using the following command:
+
+    sudo xfs_growfs /dev/vdb
+
+Once you are done with your operations and you want to detach the volume, please remember to unmount the volume before detaching it!
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:211]: This docs is only about enlargement, so I think it does not fit this part. Please remove it.

File: docs/cloud/pouta/persistent-volumes.md [hunk 5]
```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
...
+Once you have logged in to your virtual machine, you can list the
+volumes:
+
+    sudo parted -l
+
+Similar to the previous persisitent volume creation you can identify the volume based on its size. First mount the path:
+
+    sudo mount /dev/vdb /media/volume
+
+Now you can check the status of the volume in a mounted XFS filesystem using the following command:
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:207]: Please check that you need to mount the volume before expanding it. It should be safer to expand an unmounted volume.

**Review by lvarin — CHANGES_REQUESTED** [review]
Minor typos, and small details.
Fix the changes and I will merge it.

File: docs/cloud/pouta/persistent-volumes.md (hunk 4 above)
Comment (jemaltahir) [reply docs/cloud/pouta/persistent-volumes.md:211]: How can i remove it from here?

File: docs/cloud/pouta/persistent-volumes.md (hunk 5 above)
Comment (jemaltahir) [reply docs/cloud/pouta/persistent-volumes.md:207]: yes we need it.

Comment (jemaltahir) [reply docs/cloud/pouta/persistent-volumes.md:207]: `[cloud-user@tahir-jemal-volume-test ~]$  sudo xfs_growfs /dev/vdc
xfs_growfs: /dev/vdc is not a mounted XFS filesystem
[cloud-user@tahir-jemal-volume-test ~]$ sudo mount /dev/vdc /media/volume
[cloud-user@tahir-jemal-volume-test ~]$  sudo xfs_growfs /dev/vdc
meta-data=/dev/vdc               isize=512    agcount=20, agsize=655360 blks
         =                       sectsz=512   attr=2, projid32bit=1
         =                       crc=1        finobt=0 spinodes=0
data     =                       bsize=4096   blocks=13107200, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
log      =internal               bsize=4096   blocks=2560, version=2
         =                       sectsz=512   sunit=0 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
data blocks changed from 13107200 to 26214400
[cloud-user@tahir-jemal-volume-test ~]$ 
`

File: docs/cloud/pouta/persistent-volumes.md [hunk 6]
```diff
@@ -183,3 +183,28 @@ colleague to whom you want to transfer the volume.
 Your colleague can accept the transfer request of this volume:
 
     openstack volume transfer request accept <transferID> <authKey>
+
+## Expanding size to the attached volumes in the Pouta web interface
+
+Previuosly you have created and attached a volume. In this section you are going to enlarge the size of the volume attached to the instance. Before you attempt for volume expansion you have to detach the volume from the instance, please remember to unmount the volume before detaching it!
```
Comment (lvarin) [inline docs/cloud/pouta/persistent-volumes.md:189]:
```suggestion
Previously you have created and attached a volume. In this section you are going to enlarge the size of the volume attached to the instance. Before you attempt for volume expansion you have to detach the volume from the instance, please remember to unmount the volume before detaching it!
```

**Review by lvarin — APPROVED** [review]
LGTM, thanks for the changes

**Pull request merged**